### ISSUE TITLE
Support for Python and PHP docs

### ISSUE THREAD

**Issue by jlecordier** [mdesantis/force-english-content#88]
Hi, thanks for this useful extension :)

I'd like to know if you would be interested in supporting the Python and PHP docs ?

If you are opened to this addition, but are not interested in developing the feature, I could do a Pull Request in an hypothetical future 😉

**Comment by jlecordier** [comment]
- Python :
  - Reference :
    - French : https://docs.python.org/fr/3/reference/import.html
    - English : https://docs.python.org/3/reference/import.html
  - Library :
    - French : https://docs.python.org/fr/3/library/datetime.html
    - English : https://docs.python.org/3/library/datetime.html
  - In short :
    - lang : https://docs.python.org/{lang}/3/{path}.html
    - English : https://docs.python.org/3/{path}.html
- PHP :
  - Language :
    - French : https://www.php.net/manual/fr/language.namespaces.definition.php
    - English : https://www.php.net/manual/en/language.namespaces.definition.php
  - Function :
    - French : https://www.php.net/manual/fr/function.count.php
    - English : https://www.php.net/manual/en/function.count.php
  - In short :
    - lang : https://www.php.net/manual/{lang}/{path}.php
    - English : https://www.php.net/manual/en/{path}.php

**Comment by mdesantis** [comment]
@jlecordier I have a question, I can reproduce the unfriendly redirection with PHP:

```
curl -s -L -H 'Accept-Language: fr' -XGET "https://www.php.net/manual/language.namespaces.definition.php"
> French content
```

But I can't with Python docs:

```
curl -s -L -H 'Accept-Language: fr' -XGET "https://docs.python.org/3/reference/import.html"
> English content
```

What is your use case for forced redirection with Python?

**Comment by jlecordier** [comment]
When I am using a search engine, results are often linking to a localized page. So I have to manually change the language back to english everytime :(

**Comment by mdesantis** [comment]
Makes sense to me ;)

**Comment by jlecordier** [comment]
It appears that in the PHP docs, if one tries to manually select an other language, it completely messes the request up.

Example :
1. Go to : https://www.php.net/manual/en/function.array-keys.php
2. Select French in the `Change language` selector
3. Sometimes, it shows a nginx error with the following URL : https://www.php.net/manual/en/change.php?page=fr%2Ffunction.array-keys.php

I can't reproduce the bug when the extension is turned off

**Comment by jlecordier** [comment]
Other than that, it looks great ! Thank you so much !

**Comment by mdesantis** [comment]
> It appears that in the PHP docs, if one tries to manually select an other language, it completely messes the request up.
> 
> Example :
> 
>     1. Go to : https://www.php.net/manual/en/function.array-keys.php
> 
>     2. Select French in the `Change language` selector
> 
>     3. Sometimes, it shows a nginx error with the following URL : https://www.php.net/manual/en/change.php?page=fr%2Ffunction.array-keys.php
> 
> 
> I can't reproduce the bug when the extension is turned off

It seems that the `https://www.php.net/manual/change.php` doesn't support the `en` path fragment. I need to exclude that from the rewrite logic. Nice catch!

**Comment by mdesantis** [comment]
@jlecordier hey there! can you update to version `3.0.6` and check that it works properly now? Thanks!

**Comment by jlecordier** [comment]
I don't know what "work properly" exactly means, but the requirements I have written on this issue are fulfilled ! 😜

I love your coding style btw 😉
